components/ca-crud.html

```diff
@@ -704,7 +716,9 @@
             if (previewable) {
                 arr.splice(0, 0, 'Preview');
             }
-            const buttons = Object.keys(labelToAction).concat(arr);
+
+            // add the action and navigation buttons to the base buttons for the dialog
+            const buttons = Object.keys(labelToAction).concat(Object.keys(labelToNav)).concat(arr);
```

Comment:
I would say that if a schema contained both an "action:users" and a "nav:users" then the thing that produced the schema is at fault and needs changing. The CRUD component is dumb and simply adds buttons according to the schema and so I think is correct in this instance